src/rules/quotemarkRule.ts

```diff
@@ -155,6 +155,10 @@ function walk(ctx: Lint.WalkContext<Options>) {
                 }
             }
 
+            if (actualQuoteMark === fixQuoteMark) {
```

Comment:
Does the `this.toastCtrl.present(`Please tick "Yes, I'm sure" to confirm`);` from the issue not work as a test?